homeassistant/components/iskra/sensor.py

```diff
@@ -205,6 +247,19 @@ async def async_setup_entry(
             if description.key in sensors
         )
 
+        if device.supports_counters:
+            for index, counter in enumerate(device.counters.non_resettable[:4]):
```

Comment:
What will the unique_id be?

Reply:
unique_id is assembled in entity.py using device serial and entity description key. entity description key is ATTR_RESETTABLE_COUNTER or ATTR_NON_RESETTABLE_COUNTER with added counter index

Reply:
i hope this is OK.

Reply:
Can the counter change? We want to avoid setting up and having the data point that previously represented counter `1` to be counter `2` and vice versa

Reply:
they cannot those counters are directly from the energy meter's registers, always in same order